package.json

```diff
@@ -1012,14 +1012,15 @@
         "all": "npm i && npm run lint && npm test"
     },
     "devDependencies": {
+        "@azure/arm-resources": "^4.0.0",
         "@azure/cosmos": "^3.6.3",
         "@types/documentdb": "^1.10.2",
         "@types/fs-extra": "^8.1.0",
         "@types/keytar": "4.0.1",
         "@types/mocha": "^7.0.2",
         "@types/mongodb": "^3.3.2",
-        "@types/node": "^12.0.0",
-        "@types/vscode": "1.40.0",
+        "@types/node": "^14.14.5",
```

Comment:
You should keep the Node types at 12. Our version of Node is controlled by VS Code. You can find that info by doing "Code -> About Visual Studio Code" (the minor/patch version doesn't really matter, just the major version). For example:
<img width="427" alt="Screen Shot 2020-10-27 at 3 27 19 PM" src="https://user-images.githubusercontent.com/11282622/97369022-557fbe80-1869-11eb-96d5-1c0ff53b2a0b.png">